core/src/ids.rs

```diff
@@ -243,6 +247,23 @@ impl ReservationId {
     }
 }
 
+/// A builtin actor type identifier. Serves as a seed for a builtin actor ID generation.
+#[derive(Clone, Copy, Debug, Decode, Encode, Eq, PartialEq, TypeInfo)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
does it really need to derive serde traits? If I'm not mistaken that functionality isn't used

Reply:
It should be required by some trait bound - otherwise I wouldn't have made it derive those traits. But I'll double check.